lib/ansible/galaxy/collection/__init__.py

```diff
@@ -116,6 +116,11 @@
     CollectionDependencyInconsistentCandidate,
 )
 from ansible.galaxy.dependency_resolution.versioning import meets_requirements
+from ansible.galaxy.collection.gpg import (
```

Comment:
Mind moving this above? Lines 100-107 contain all the `ansible.galaxy.collection` imports, let's keep them sorted for consistency.